lua/weapons/gmod_tool/stools/starfall_screen.lua

```diff
@@ -1,4 +1,5 @@
-TOOL.Category		= "Wire - Display"
+TOOL.Category		= "Visuals/Screens"
+TOOL.Wire_MultiCategories = { "Chips, Gates" } -- Also add this to the chips, gates category so that it can be found near the processor
```

Comment:
Well, you set TOOL.Category to "Chips, Gates" for the processor, which is the only entry in TOOL.Wire_MultiCategories for the screen. TOOL.Wire_MultiCategories is not even set for the processor, though and TOOL.Category for the screen is set to "Visuals/Screens".

I think they are quite different... is it intentional, though? Why?